spec/clearance/back_door_spec.rb

```diff
@@ -46,6 +46,18 @@
     end
   end
 
+  it "strips 'as' from the params" do
```

Comment:
Is it worth adding another test case here where we have multiple query params? I think the current test would pass if we deleted all the query params.

Reply:
Resolved in 52d7d94

Reply:
Looks good to me!